qiskit/scheduler/methods/basic.py

```diff
@@ -157,16 +157,19 @@ def get_measure_schedule() -> CircuitPulseDef:
             default_sched = inst_map.get('measure', qubits)
             for time, inst in default_sched.instructions:
```

Comment:
awesome job with this section, its tricky. 👍 This is a little nitpick: if you pull sched._union(...) out, you can use the if/else just to figure out the value of `mem_slot`. It would highlight the purpose of the if/else